packages/wallet-frontend/src/components/Purses.jsx

```diff
@@ -29,26 +30,21 @@ export default function Purses() {
       <Typography variant="h6">Purses</Typography>
       {Array.isArray(purses) && purses.length > 0 ? (
         <List>
-          {purses.map(
-            ({ pursePetname, brandRegKey, issuerPetname, extent }) => (
-              <ListItem key={pursePetname} value={pursePetname} divider>
-                <ListItemIcon className={classes.icon}>
-                  <PurseIcon />
-                </ListItemIcon>
-                <ListItemText
-                  primary={pursePetname}
-                  secondary={
-                    <>
-                      <b>
-                        {JSON.stringify(extent)} {issuerPetname}
-                      </b>{' '}
-                      {brandRegKey ? <i>({brandRegKey})</i> : ''}
-                    </>
-                  }
-                />
-              </ListItem>
-            ),
-          )}
+          {purses.map(({ pursePetname, currentAmount }) => (
+            <ListItem key={pursePetname} value={pursePetname} divider>
+              <ListItemIcon className={classes.icon}>
+                <PurseIcon />
+              </ListItemIcon>
+              <ListItemText
+                primary={pursePetname}
+                secondary={
+                  <>
```

Comment:
You don't need the `<>` and `</>` here... they're just for fragments that contain more than one element.

Reply:
Ok, thanks!